spec/defines/admin_spec.rb

```diff
@@ -11,6 +11,7 @@
           {
             pulp_settings: '/etc/pulpcore/settings.py',
             user: 'pulpcore',
+            working_dir: '/var/lib/pulp'
```

Comment:
Hey @ianballou , could you please add to the test below, under `is_expected.to contain_exec('pulpcore-manager help')`,

`with_cwd('/var/lib/pulp')`

This way we are not only passing the param from the parent module which `pulpcore::admin` needs, we are also testing that the feature works properly at the business logic level (puppet catalogue) and continues working as expected with any future changes.